content/blog/2019-01-31-rdhs.md

```diff
@@ -0,0 +1,787 @@
+
+This is all to the say that the DHS provides really useful data. However, although
+standard health indicators are routinely published in the survey final reports 
+that are published by the DHS programme, much of the value of the
+DHS data is derived from the ability to download and analyse the raw
+datasets for subgroup analysis, pooled multi-country analysis, and extended 
+research studies. 
+
+This where I got involved, in trying to create a tool that helped enable 
+researchers to quickly gain access to the raw data sets. 
+
+#### How did I get involved
+
+I am fortunate enough to be a PhD student in a really large department at 
+Imperial College London, which means that I get the opportunity to be 
+involved in many projects that are outside the scope of my actual PhD. 
+The "downside" of that is sometimes you get given "code monkey" jobs as the 
+bottom rung of the monkey ladder. And so, a few months into my PhD (Nov 2016), 
+I was given the job of downloading data on malaria test results from 
+the DHS programme that was going to be used by some collaborators. 
+At the time I was very happy to be involved, however, I was 
+apprehensive to spend too long on the job as I didn't know how much time to be 
+spending on side projects vs my PhD (something I still don't know with 6 months
+to go). This combined with only having a year or so's experience writing R meant
+that the code I wrote to do the job was a bunch of scrappy scripts that required
+manually downloading the datasets before parsing them with these R scripts. Dirty
+but it got the job done. 
+
+Some time passed, and another collaborator wanted some different data collated
+from the DHS programme. At this point, I had 6 more months familiarity with 
+R and knew a bit more so I started writing it as an R package. However, it was 
+still messy and it required manually downloading the datasets first, but I was 
+happy with it and again it wasn't a major project of mine. This would have been
+probably where the project ended if I hadn't had a conversation (Sept 2017)
+in the tea room (prompted solely by the presence of free biscuits) with the 
+other main author of `rdhs`, [Jeff Eaton](https://twitter.com/eatonjw). 
+
+We got chatting, and realised we both had a bunch of scripts for doing bits of
+the analysis pipeline. We also realised that we had both had numerous requests 
+for data sets from the DHS programme at which point we thought it would be best
+to do something properly. I had also at this point been keen to start using `testhat`
+within my work as I had been told it would save me time in the future, and up till
+that point I hadn't found a good case to get to grips with it (mainly writing code
+on my own, that was never very big and was only used by myself). And so we started
+writing `rdhs`, which was accepted by `ropensci` and `CRAN` in December 2018.
+
+### Package overview
+
+>
+Disclaimer: A lot of the following section (the **API** and **Dataset Downloads**
```

Comment:
Rather than replicating the vignette contents here, perhaps you could list what is covered in the vignette and link to it. In the post you can narrate your way briefly through the intriguing example without giving all the details so that people can see the potential of the package. 

this is good: "For example, to find out the trends in antimalarial use in Africa, and see if perhaps antimalarial prescription has decreased after RDTs were introduced (assumed 2010)." with the plot.

and this is a good end of that section:
"We now have managed to go from our opening question [make sure this is crystal clear at beginning of example] to a finalised data set that we can use going forwards for any downstream analysis (and hopefully it didn’t take that long to do it!)"